torch/_subclasses/fake_utils.py

```diff
@@ -9,17 +10,53 @@
 aten = torch.ops.aten
 
 
+def has_storage(t):
+    try:
+        t.storage()
+        return True
+    except Exception:
+        return False
```

Comment:
nyeh. This is going to trigger exceptions lol

Reply:
i looked into expose `has_storage` as a method on python tensor but i was a little confused about `metaclass_properties`. where should I put it ?

Reply:
i guess `THPDtype_properties`..

Reply:
you could bind it as a free standing function in _C, that's how we exposed a lot of "private" tensor internals